drivers/mrf24j40/mrf24j40.c

```diff
@@ -156,7 +156,7 @@ void mrf24j40_tx_exec(mrf24j40_t *dev)
      */
     mrf24j40_reg_write_long(dev, MRF24J40_TX_NORMAL_FIFO, dev->header_len);
 
-    if (dev->netdev.flags & NETDEV_IEEE802154_ACK_REQ) {
+    if (dev->fcf_low & IEEE802154_FCF_ACK_REQ) {
```

Comment:
The kw2xrf driver compares both the device flags and the frame header bits before deciding on waiting for ack or not. 

RFC: Does it make sense to allow frames with the ACK_REQ bit set to be sent without waiting for the ACK, like the kw2xrf driver does, or is the device flag only meant for the link layer implementation to decide if the frame header bit should be set and the device driver only follows the frame content?

Reply:
> RFC: Does it make sense to allow frames with the ACK_REQ bit set to be sent without waiting for the ACK, like the kw2xrf driver does, or is the device flag only meant for the link layer implementation to decide if the frame header bit should be set and the device driver only follows the frame content?

For drivers I would follow the frame FCF flags. Main reason is that the receiving radio will check this same field to decide whether to ack a frame, so to me it only makes sense to ensure that both the sending and receiving hardware match behaviour and expectation.